rsconnect/api.py

```diff
@@ -16,6 +16,7 @@
 import typing
 import webbrowser
 from os.path import abspath, dirname
+from ssl import SSLError
```

Comment:
`ssl.SSLError` is the same object as `_ssl.SSLError`.

See:

https://github.com/python/cpython/blob/v3.8.0/Lib/ssl.py#L102-L105

https://github.com/python/cpython/blob/v3.12.3/Lib/ssl.py#L104-L107See